makefile.lst

```diff
@@ -226,11 +226,13 @@ CORE.SERVICES       := compression            \
 # Dependencies between oneAPI and core (CPU-only) algorithms
 ONEAPI.ALGOS.pca := CORE.pca
 ONEAPI.ALGOS.svm := CORE.svm
+ONEAPI.ALGOS.decision_forest := CORE.decision_forest
 
 # List of algorithms in oneAPI part
 ONEAPI.ALGOS := \
     pca         \
-    svm
+    svm         \
+    decision_forest
```

Comment:
I think it's better to add alphabetically

Reply:
fixed